### ISSUE TITLE
PanelMenu: Different item is focussed when item is pressed

### ISSUE THREAD

**Issue by vdbongard** [primefaces/primeng#19526]
### Describe the bug

If a PanelMenu item is actively being selected, another item in the PanelMenu group is focused as well.

### Reason for not contributing a PR

- [ ] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

primeng.org/panelmenu#template

### Environment

Chrome 146, Windows 11

### Angular version

21.2.6

### PrimeNG version

v21

### Steps to reproduce the behavior

1. Go to the "Template" section of the PanelMenu docs ([primeng.org/panelmenu#template](https://primeng.org/panelmenu#template))
2. Open the "Mail" group
3. Actively select the second entry of "Inbox" (hold the mouse down while clicking the item)
4. The first item "Compose" also gets highlighted, while pressing the second item "Inbox"

<img width="900" height="569" alt="Image" src="https://github.com/user-attachments/assets/cc33a6c7-2c35-43c9-9b44-7eb92bc9fc47" />

### Expected behavior

The first item should not get highlighted while pressing the second item.

**Comment by cetincakiroglu** [comment]
Fixed in PrimeNG v22
